platform/public/demo.html

```diff
@@ -0,0 +1,6 @@
+<html>
```

Comment:
What is this all about?

Reply:
It is just a demonstration that one can server up additional files coming from a specified directory.  That allows us to split off static files if we want, and have the directory get included automatically.

Reply:
A place to put demo loads/alternate configs etc.